packages/core/src/app/shipping/AllocateItemsModal.tsx

```diff
@@ -103,8 +105,21 @@ const AllocateItemsModal: FunctionComponent<AllocateItemsModalProps & FormikProp
 
     const modalFooter = (
         <>
-            <Button onClick={onRequestClose} variant={ButtonVariant.Secondary}><TranslatedString id="shipping.multishipping_items_allocate_cancel" /></Button>
-            <Button disabled={!dirty} onClick={submitForm} type="submit" variant={ButtonVariant.Primary}>{hasItemsAssigned ? <TranslatedString id="shipping.multishipping_items_allocate_save" /> : <TranslatedString id="shipping.multishipping_items_allocate_allocate" />}</Button>
+            <Button onClick={onRequestClose} variant={ButtonVariant.Secondary}>
+                <TranslatedString id="shipping.multishipping_items_allocate_cancel" />
+            </Button>
+            <Button
+                disabled={!dirty}
+                isLoading={isLoading}
```

Comment:
Should we disable the cancel button while loading?

Reply:
Good catch. Disabled it when loading [here](https://github.com/bigcommerce/checkout-js/pull/2107/commits/cd70595da7298cf5190c7c7930f62411d31b32fc)